packages/wallet/src/data/services/market-proxy/index.ts

```diff
@@ -0,0 +1,688 @@
+/**
+ * @see https://docs.coingecko.com/ for CoinGecko API documentation
+ * @see https://test.market.status.im for proxy endpoint
+ *
+ * This module uses CoinGecko API through a proxy at https://test.market.status.im
+ * Example: https://pro-api.coingecko.com/api/v3/coins/list -> https://test.market.status.im/v1/coins/list
+ */
+
+// import 'server-only'
+
+import { serverEnv } from '../../../config/env.server.mjs'
+import erc20TokenList from '../../../constants/erc20.json'
+import { markApiKeyAsRateLimited } from '../api-key-rotation'
+
+import type {
+  CoinGeckoCoinDetailResponse,
+  CoinGeckoCoinHistoryResponse,
+  CoinGeckoCoinListResponse,
+  CoinGeckoMarketChartResponse,
+  CoinGeckoSimplePriceResponse,
+} from './types'
+
+const PROXY_BASE_URL = 'https://test.market.status.im'
```

Comment:
What's the cache config/ttl on these proxies please? Any custom config per rpc method or market endpoint?